.travis.yml

```diff
@@ -12,10 +12,26 @@ addons:
 cache:
   directories:
     - node_modules
-services:
-  #- docker
 before_script:
-  #Docker setup goes here
   - pwd
   - cp ~/build/HackIllinois/api-2017/config/test.config.template ~/build/HackIllinois/api-2017/config/test.config
   - npm install
+before_deploy:
+  - pwd
+  - cd /tmp/
+  - wget https://repo1.maven.org/maven2/org/flywaydb/flyway-commandline/4.0.3/flyway-commandline-4.0.3-linux-x64.tar.gz && tar -xzf flyway-commandline-4.0.3-linux-x64.tar.gz && mv flyway-4.0.3 /opt/flyway-4.0.3 && ln -s /opt/flyway-4.0.3/jre/bin/java /usr/local/bin/java && ln -s /opt/flyway-4.0.3/flyway /usr/local/bin/flyway && rm -rf /tmp/flyway-*
+deploy:
+  provider: elasticbeanstalk
+  skip_cleanup: true
+  access_key_id:
+    secure: "F8tYUsTeexOsO+VMj7hsk55xkOCt/wwN5kp8iV2vjhOm9XNawP08e/bzzLTPMCqveZbhaoTEO4qKnQ6g8Eeyp97oPS/53Y2zlwS/qQ3qP0W/ggSyTIc8ed8a+o4R5Bx9sLcJr5Qt7unTNjmIJY5kJUe8f8SQyeBEFJ/Y9NtYHt4dq5WA2mKGRN0+j7aN5mypiBoHdS7fAnhyvk+qgfZMa1DWgytXn3i7AHnONhoRbPxwzk8SaGQTfZQGHoqrjiHavwVL/X2qLz/7GEOgKkdRluOI+ROgwIdSwRBNsoS308oA3XN0zLT/40IQ7Ct/I6D7oQrwILN8GOBbtLYrQuv54HXHfXJg6ESer+32ryyrqdQFZuxejQ5VMbD7JZUWlZhrojk6dxc9ETHSC3cKBA+kHJnvkFLHvtR/DYR7R7qDalN6QcLvUYp4PGZqWcdrMJlt5Kb6t0q1lK4CIHX/fcfViKbV1HnjL1a6edyoGqdQvxpSRFzz+znrpoh3ZI6h5Th4AknrTBRaczCaNyVnXuBtz/adjH2IqtydulKQm4tBJoFT+9HWQKJRcxFvRP2J8rnaPD8YVUJRAW4NDWVepkWUne8ZtWn7LHWQrgeBi4oxS9MW8jGXTsUCZSpyG7/aB86ZhHm5FwTdzVY0q0edyq7ikKZEBjSjKqBbpslYYoVEZvM="
+  secret_access_key:
+    secure: "bCSmJe3dcBMDwDXzK7HWhkSJPNOrHDbv6uAKv8yD/Ny6Qo+EfJdw81ygGvKBHepI+dj3fPnDAZZBUruRzzLqMFMMzzQv/cunZhM+wFyjTZP4KP2iednk1MANdzivkfJj/BoyfUD+c5OOr5OK83zHgQomsv35XnwF9v7TCbz59fN/jwN15fU+Y161rtSoU/8JutWAQ2oQzXbRkXu9zr1tPkeT4wjjl/+wCfplLUWocwQx2OA+VJI66tTuURzwtLtnZ8zLDbyukY+kG41b8dRdx5agdD4CT8QytSst527zVxualYK3Mhjp4T79CDXZMOfn53xPK2H3gklnca+H+mjple5HS/i6aItfYoqOqCqFf2EiJ7rFliNP0tCvf3eKJIsGPPvMaqoRRep7ZJmyTDvIg9WyjqckXtYY/ck1FuvwdcibbEpvQD/NrrkFjVhTukxbdMhIIjYg1FkPic1q5d0AOm8txgWhKgu1tP1tzyOK/4qmC3PtVG4kKH+J1P7pzztxPTFEh5CNvdbfI/Qt+94QncFu0vgrI9vwnch24Jz7t+zhVCmLCgF1FYp915SisaTlBN+RQ0sdI/0kbd58Mb7co87psPVULPhVEgz66L/TJSXIfGWXfteRpNqCyH6upKICfg7b7Uy7M0M5oCMn8QiJyA5LBqOtbVvpkYlwlzjPcZE="
```

Comment:
They're the access keys for AWS, but encrypted with travis 
https://docs.travis-ci.com/user/encryption-keys/